### PR TITLE
fix projectEdit and projectNew style

### PR DESCRIPTION
**Pull request by IsraaSulaiman**
Relates #183

### CODE REVIEW

File: client/src/components/Projects/ProjectEdit/index.jsx [hunk 1]
```diff
@@ -20,14 +21,20 @@ export default class index extends Component {
...
-    this.setState({
-      project: projectInf,
-      newProject: {
-        name: projectInf.name,
-        dsescription: projectInf.dsescription
-      }
-    });
+    const projectId = this.props.match.params.projectId;
+    if (Number.isInteger(parseInt(projectId))) {
+      const projectInf = require("./../../commonComponents/TableMember/projects.json")[0];
```
Comment (tomduggan85) [inline client/src/components/Projects/ProjectEdit/index.jsx]: @IsraaSulaiman I think for organizing the code, it's best if the imports are all at the top of the file, together - I'd recommend moving this one up to the top of the file (and then still using it here).

File: client/src/components/Project/index.jsx [hunk 2]
```diff
@@ -17,6 +18,13 @@ class Scrums extends Component {
   }
 
   componentDidMount() {
+    const projectId = this.props.match.params.projectId;
+    const scrumId = this.props.match.params.scrumId;
+    if (Number.isInteger(parseInt(projectId)) && Number.isInteger(parseInt(scrumId))){
```
Comment (ashatat) [inline client/src/components/Project/index.jsx]: I don't understand what is the purpose of this `if` statement :thinking:

Comment (IsraaSulaiman) [reply client/src/components/Project/index.jsx]: That's to make sure the scrumId is a number, not a string. without this validation, the app will crash if you tried to fetch a URL like `/projects/1/israa`

Comment (ashatat) [reply client/src/components/Project/index.jsx]: I don't think it would crash, it'd just return an empty array

**Pull request merged**